src/src/CustomBuild/ESPEasyLimits.h

```diff
@@ -72,19 +82,15 @@
 // ***********************************************************************
 #ifndef DEVICES_MAX
   // TODO TD-er: This should be set automatically by counting the number of included plugins.
-  #if defined(PLUGIN_BUILD_TESTING) || defined(PLUGIN_BUILD_DEV)
-    # define DEVICES_MAX                      95
-  #else    // if defined(PLUGIN_BUILD_TESTING) || defined(PLUGIN_BUILD_DEV)
-    # ifdef ESP32
-      # ifdef PLUGIN_BUILD_MAX_ESP32
-        #  define DEVICES_MAX                    135
-      # else // ifdef PLUGIN_BUILD_MAX_ESP32
-      #  define DEVICES_MAX                      100
-      # endif // ifdef PLUGIN_BUILD_MAX_ESP32
-    # else // ifdef ESP32
+  # ifdef ESP32
+    # define DEVICES_MAX                      130
```

Comment:
This can be reset to 135, I assume, we're approaching 130 plugins already 😉

Reply:
Why not set it to 255 for ESP32?

Reply:
There are only 2 arrays reserved for that size, 1 of `bool` and 1 of `pluginID_t` (==`uint8_t`), so that shouldn't be an issue, indeed.
Let's do it, the next step, going beyond 255 is much more work 🤣